pyhf/commandline.py

```diff
@@ -25,3 +27,18 @@ def xml2json(entrypoint_xml, basedir, output_file, track_progress):
     else:
         json.dump(spec, open(output_file, 'w+'), indent=4, sort_keys=True)
         log.info("Written to {0:s}".format(output_file))
+
+@pyhf.command()
+@click.argument('workspace', default = '-')
+@click.option('--output-file', help='The location of the output json file. If not specified, prints to screen.', default=None)
+def cls(workspace,output_file):
+    specstream = click.open_file(workspace)
+    d = json.load(specstream)
+    p = Model({'channels':d['channels']}, poiname=d['toplvl']['measurements'][0]['config']['poi'])
+    result = runOnePoint(1.0, d['data']['channel1'] + p.config.auxdata, p)
```

Comment:
this is hardcoded to take the 'first' measurement. in my experience not a lot of analyses actually use the measurement feature all that much..99% probably only have one

Reply:
Should spit a warning if more than one `poiname` detected -- and also do some logging to indicate which one is being used.